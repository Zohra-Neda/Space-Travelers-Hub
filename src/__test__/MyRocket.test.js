import React from "react";
import renderer from 'react-test-renderer'
import MyRocketCard from "../components/MyRocketCard";

describe('MyRocketCard component', () => {
    test('renders correctly', () => {
        const mockRocket = {
            id: 'rocket1',
            name: 'Rocket1',
            description: 'This is the rocket description',
            joined: true,
        };

        const component = renderer.create(<MyRocketCard rocket={mockRocket} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapShot();
    });

})