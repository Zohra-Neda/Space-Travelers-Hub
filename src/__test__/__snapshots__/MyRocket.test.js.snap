// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyRocketCard component renders correctly 1`] = `
<div
  className="my-rocket-card border border-1 py-5 text-2xl ps-4 "
>
  Rocket1
</div>
`;
